.github/workflows/rust.yml

```diff
@@ -39,11 +39,18 @@ jobs:
     - name: Run tests
       run: |
         pushd borsh
+        echo 'cargo test --no-run ...'
         cargo test --no-run
+        echo 'cargo test ...'
         cargo test
+        echo 'cargo test --no-default-features ...'
         cargo test --no-default-features
-        cargo test --all-features
+        echo 'cargo test --no-default-features --features hashbrown,rc ...'
+        cargo test --no-default-features --features hashbrown,rc
+        echo 'cargo test --features rc ...'
+        cargo test --features rc
         popd
+        echo 'cargo test --workspace ...'
```

Comment:
Let's use [`set -x`](https://www.educative.io/answers/bash--x-and-set--x) at the beginning of the `run` command, so we don't need to manually write `echo` commands.